x11-misc/shutter/shutter-0.96.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop optfeature prefix xdg
+
+DESCRIPTION="Feature-rich screenshot program"
+HOMEPAGE="https://shutter-project.org/"
+SRC_URI="https://github.com/shutter-project/shutter/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-lang/perl
```

Comment:
Check if it installs Perl modules (into perl5/vendor_perl/5.xx/) or libraries linked to libperl.so. If yes, then add := operator...

Reply:
It does not